Library/Homebrew/dev-cmd/bump.rb

```diff
@@ -38,23 +40,44 @@ def bump
       raise UsageError, "`--limit` must be used with either `--formula` or `--cask`."
     end
 
-    formulae_and_casks = if args.formula?
-      args.named.to_formulae.presence
-    elsif args.cask?
-      args.named.to_casks.presence
-    else
-      args.named.to_formulae_and_casks.presence
-    end
+    formulae_and_casks =
+      if args.formula?
+        args.named.to_formulae
+      elsif args.cask?
+        args.named.to_casks
+      else
+        args.named.to_formulae_and_casks
+      end&.sort_by do |formula_or_cask|
+        formula_or_cask.respond_to?(:token) ? formula_or_cask.token : formula_or_cask.name
+      end
 
     limit = args.limit.to_i if args.limit.present?
 
     if formulae_and_casks
       Livecheck.load_other_tap_strategies(formulae_and_casks)
 
+      ambiguous_casks = []
+      if !args.formula? && !args.cask?
+        ambiguous_casks = formulae_and_casks
+                          .group_by { |item| Livecheck.formula_or_cask_name(item, full_name: true) }
+                          .select { |_name, items| items.length > 1 }
+                          .values.flatten
```

Comment:
```suggestion
                          .values
                          .flatten
```

but, given you don't use the values for the `.select` you could use `.values` before that.